s3repo/model.py

```diff
@@ -81,22 +104,28 @@ def __init__(self, s3_settings):
         self.sync_thread.start()
 
     @staticmethod
-    def _format_paths(dist_path, dist_version, dist_base, filename, product):
+    def _format_paths(dist_path: str, dist_version: str, dist_base: str,
+                      filename: str, product: str) -> List[Tuple[str, str]]:
         """Formats the file path and repository path according
         to the filename and distribution information.
         Returns an array of tuples (repo_path, path).
         """
-        result = []
-        file_type_err = 'The "{0}" file does not match the type of files ' +\
-            'used in the {1}-based repositories.'
+        result: List[Tuple[str, str]] = []
+        file_type_err = 'The "{0}" file does not match the type of files ' + \
```

Comment:
Not related with the commit.
What's the code style rule being violated here?

Reply:
rolled back